protocol/x/vault/types/params_test.go

```diff
@@ -80,6 +80,30 @@ func TestValidateQuotingParams(t *testing.T) {
 			},
 			expectedErr: types.ErrInvalidActivationThresholdQuoteQuantums,
 		},
+		"Failure - SkewFactorPpm is high. Product of SkewFactorPpm and OrderSizePctPpm is above threshold.": {
```

Comment:
nit: Add a test where skew and order size are max uint32 to make sure no overflow errors.